src/Component/Symfony/Routing/Factory/DeleteOperationRoutePathFactory.php

```diff
@@ -28,10 +29,7 @@ public function createRoutePath(HttpOperation $operation, string $rootPath): str
         $identifier = $operation->getResource()?->getIdentifier() ?? 'id';
 
         if ($operation instanceof DeleteOperationInterface) {
-            $path = match ($shortName) {
-                'delete' => '',
-                default => '/' . $shortName,
-            };
+            $path = $operation instanceof ApiOperationInterface && 'delete' === $shortName ? '' : '/' . $shortName;
```

Comment:
What about adding the `delete` string injected in this service, allowing developers to customize the url slug can maybe be a thing.

Reply:
It was just an idea, it will need a dedicated PR